teams/project/builds-tools.yml

```diff
@@ -0,0 +1,36 @@
+# yaml-language-server: $schema=../sub-teams-schema.json
+name: builds-tools
+description: Team to work on conda-build and other build tools in the conda ecosystem
+charter: project
+requirements: null
+scopes:
+    codeowners:
+        - conda-incubator/conda-recipe-manager
+        - conda/conda-build
+    other:
+        Matrix (public): https://matrix.to/#/#conda-build-tools:matrix.org
+links:
+    - https://github.com/conda/conda-build/issues/4698
+    - https://github.com/conda/governance/issues/128
+members:
+    beeankha: https://github.com/conda/conda-build/issues/4698
+    beckermr: null
+    chenghlee: https://github.com/conda/conda-build/issues/4698
+    dbast: https://github.com/conda/conda-build/issues/4698
+    dholth: https://github.com/conda/conda-build/issues/4698
+    isuruf: https://github.com/conda/conda-build/issues/4698
+    jaimergp: https://github.com/conda/conda-build/issues/4698
+    jakirkham: https://github.com/conda/conda-build/issues/4698
+    JeanChristopheMorinPerso: https://github.com/conda/conda-build/issues/4698
+    jezdez: https://github.com/conda/conda-build/issues/4698
+    kenodegard: https://github.com/conda/conda-build/issues/4698
+    marcelotrevisani: https://github.com/conda/conda-build/issues/4698
+    mbargull: https://github.com/conda/conda-build/issues/4698
+    msarahan: https://github.com/conda/conda-build/issues/4698
+    ryanskeith: https://github.com/conda/conda-build/issues/4698
+    travishathaway: https://github.com/conda/conda-build/issues/4698
+    wolfv: https://github.com/conda/conda-build/issues/4698
+    schuylermartin45: https://github.com/conda/governance/issues/128
+emeritus:
+    Callek: null
```

Comment:
@Callek is no longer in the GH team but I haven't found a record of when he left the team

Reply:
@kenodegard 
I left the team Monday May 6, 2024.  (Jannis was cleaning up team memberships, and I didn't need it)

Reply:
I'm *also* ok if you don't list me as Emeritus, I never considered myself active in the "team" and my participation in our meetings, was mainly just to hear about problems and upcoming changes.